doc/news/changes/minor/20200111Bangerth

```diff
@@ -0,0 +1,6 @@
+New: The mechanism implemented by Dataout via the `first_cell()` and
```

Comment:
I am pretty sure that you need `DataOut::first_cell()` and `DataOut::next_cell()` for `doxygen` to pick it up.

Reply:
I specifically didn't want doxygen to pick these names up because I intend to remove these functions :-)

Reply:
I see. But I guess you still want 
```suggestion
New: The mechanism implemented by DataOut via the `first_cell()` and
```